crates/sui-framework/sources/math.move

```diff
@@ -22,6 +22,15 @@ module sui::math {
         }
     }
 
+    /// Return the absolute value of x - y
+    public fun abs(x: u64, y: u64): u64 {
```

Comment:
"abs" normally refers to the absolute value of one number. Maybe call this "distance" instead?

Reply:
Yrs interesting